### PR TITLE
Bank details

### PR DESCRIPTION
**Pull request by charan12345678**

### CODE REVIEW

File: app/components/organisms/LitePage/desktop.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useState, useRef, useEffect } from 'react';
+import ReactTextTransition, { presets } from 'react-text-transition';
```
Comment (tuhina2020) [inline app/components/organisms/LitePage/desktop.js:2]: Remove changes to this file. just merged them to the main branch. And please create a branch from the latest staging branch before starting to work.

File: app/components/organisms/LitePage/mobile.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React, { useState, useRef, useEffect } from 'react';
+import ReactTextTransition, { presets } from 'react-text-transition';
```
Comment (tuhina2020) [inline app/components/organisms/LitePage/mobile.js:2]: Remove changes to this file. just merged them to the main branch. And please create a branch from the latest staging branch before starting to work.

File: app/containers/ProfilePage/index.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import React, { useEffect } from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
```
Comment (tuhina2020) [inline app/containers/ProfilePage/index.js]: Always add dependencies to package.json file.

File: app/templates/ProfileDetails/editModal.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import ContactEditForm from 'components/organisms/ProfileEditForm/contact';
 import SkillEditForm from 'components/organisms/ProfileEditForm/skills';
 import ExperienceEditForm from 'components/organisms/ProfileEditForm/ExperienceEdit';
 import PortfolioEditForm from 'components/organisms/ProfileEditForm/PortfolioEdit';
+import EditBankForm from 'components/organisms/ProfileEditForm/bankdetails';
```
Comment (tuhina2020) [inline app/templates/ProfileDetails/editModal.js:9]: This is not the write file name. Please try again with the right name.

File: app/containers/ProfilePage/index.js (hunk 3 above)
Comment (charan12345678) [reply app/containers/ProfilePage/index.js]: I added the dependency locally,Installed Tabs package thorugh npm and it got added automatically in the package.json file. but just didn't push  the package.json file ,also for now Tabs are not needed ,I removed the tabs I added but forgot to remove the import statements

File: app/templates/ProfileDetails/editModal.js (hunk 4 above)
Comment (charan12345678) [reply app/templates/ProfileDetails/editModal.js:9]: Okay ,will correct it.

**Pull request closed without merging**